app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -268,6 +305,37 @@ export const ReportResidencyFrTaxesFr: React.FunctionComponent<
             </div>
             <Page510 taxes={taxes} isPrintMode={isPrintMode} />
           </Section>
+          <Section title="Source of information">
+            <div>
```

Comment:
From a UI perspective, I think it would look better using an unordered list with 3 bullet points for etrade + ECB API + alphavantage.